pkg/relayer/session/session.go

```diff
@@ -2,37 +2,34 @@ package session
 
 import (
 	"context"
+	"log"
 	blockclient "pocket/pkg/client"
 	"pocket/pkg/observable"
 	"pocket/pkg/observable/channel"
 	sessiontypes "pocket/x/session/types"
 	"sync"
 )
 
-var _ RelayerSessions = (*relayerSessions)(nil)
+var _ RelayerSessionsManager = (*relayerSessionsManager)(nil)
 
-type (
-	sessionId        = string
-	blockHeight      = int64
-	sessionsTreesMap = map[blockHeight]map[sessionId]SessionTree
-)
+type sessionsTreesMap = map[int64]map[string]SessionTree
 
-// relayerSessions is an implementation of the RelayerSessions interface.
-type relayerSessions struct {
+// relayerSessionsManager is an implementation of the RelayerSessions interface.
```

Comment:
In the interest of time, since we decided to bias towards an E2E test, can you create correspond `*_test.go` files and leave a `TODO_TEST` comment explaining that we need to add tests in the future.